src/lib/pages/common/infiniteScroll.js

```diff
@@ -0,0 +1,24 @@
+// TODO: find correct place for this function
```

Comment:
```suggestion
```

Yeah, I think you can put this file under the `services` directory along with other JS files. And change the filename from camel case to kebab case, so that: `src/lib/services/infinite-scroll.js`

Reply:
Makes sense to me. I'll move and rename the file.